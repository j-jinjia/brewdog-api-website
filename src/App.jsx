import { React } from "react";
import MainContent from "./components/container/MainContent/MainContent";
import Navigation from "./components/container/Navigation/Navigation";
import "./App.scss";
const App = () => {
  return (
    <>
      <Navigation />
      <MainContent />
    </>
  );
};

export default App;
