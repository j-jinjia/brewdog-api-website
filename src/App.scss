*{
  box-sizing: border-box;
  margin:0;
  padding:0;
  font-family: 'Courier New', Courier, monospace;
  
}

body{
  animation:animate 20s ease infinite;
@keyframes animate {
    0% {
        background-color: #6cb150;
    }
    25% {
        background-color: #00b0e0;
    }
    
    50% {
        background-color: #ffc04c;
    }
    75% {
        background-color: #ec5739;
    }
    100% {
        background-color: #6cb150;
    }
  }
}
