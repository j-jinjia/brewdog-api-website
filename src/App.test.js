import { render, screen } from '@testing-library/react';
import App from './App';

it("should match app snapshot ", ()=>{
  const {container} = render (<App/>)
  expect(container).toMatchSnapshot();

})
