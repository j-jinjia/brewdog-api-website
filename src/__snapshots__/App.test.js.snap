// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match app snapshot  1`] = `
<div>
  <article
    class="navigation"
  >
    <img
      alt="brewdog-icon"
      class="navigation__logo"
      src="brewdog-icon.png"
    />
    <h1>
      BREWDOG
    </h1>
    <h2
      class="navigation__faq"
      tabindex="0 "
    >
      HELP
    </h2>
  </article>
  <div
    class="container"
  >
    <div
      class="functionality-bar"
    >
      <form
        class="searchBar"
      >
        <label
          for="beers"
        >
          <svg
            enable-background="new 0 0 48 48"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            version="1"
            viewBox="0 0 48 48"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="#616161"
            >
              <rect
                height="17"
                transform="matrix(.707 -.707 .707 .707 -15.154 36.586)"
                width="4"
                x="34.6"
                y="28.1"
              />
              <circle
                cx="20"
                cy="20"
                r="16"
              />
            </g>
            <rect
              fill="#37474F"
              height="12.3"
              transform="matrix(.707 -.707 .707 .707 -15.839 38.239)"
              width="4"
              x="36.2"
              y="32.1"
            />
            <circle
              cx="20"
              cy="20"
              fill="#64B5F6"
              r="13"
            />
            <path
              d="M26.9,14.2c-1.7-2-4.2-3.2-6.9-3.2s-5.2,1.2-6.9,3.2c-0.4,0.4-0.3,1.1,0.1,1.4c0.4,0.4,1.1,0.3,1.4-0.1 C16,13.9,17.9,13,20,13s4,0.9,5.4,2.5c0.2,0.2,0.5,0.4,0.8,0.4c0.2,0,0.5-0.1,0.6-0.2C27.2,15.3,27.2,14.6,26.9,14.2z"
              fill="#BBDEFB"
            />
          </svg>
        </label>
        <input
          class="searchBar__input"
          name="beers"
          placeholder="Search by name"
          tabindex="1"
          type="text"
          value=""
        />
      </form>
      <div
        class="container"
      >
        <h4
          class="filters"
          tabindex="2"
        >
          Filters
        </h4>
      </div>
    </div>
    <article
      class="card-container"
    />
  </div>
</div>
`;
